### DISCUSSION TITLE
Embedding Superset Application

### DISCUSSION THREAD

**Discussion by axoxymous-coder** [apache/superset#23063] [Q&A / Help]
Is there a way to embed Apache Superset, the whole application, into another web application? I am aware that dashboards created in Superset can be embedded into another web site/application, but my requirement is that I need to embed the whole Superset app into another web application such that users can themselves create their own dashboards. Giving direct access to Superset is not an option for my use case. Any pointers or help on this will be highly appreciated.

**Comment by aanw4109** [comment]
@axoxymous-coder were you able to do this ?
